charmhelpers/contrib/openstack/context.py

```diff
@@ -1581,7 +1581,8 @@ def __call__(self):
             return {'bind_host': '0.0.0.0'}
 
 
-MAX_DEFAULT_WORKERS = 4
+MAX_DEFAULT_WORKERS = 16
+MAX_DEFAULT_WORKERS_CONTAINER = 4
```

Comment:
I think having two defaults is slightly confusing. The default is to set a max workers to some sensible value, if the operator of the charm doesn't set the worker-multiplier.  If they do, then neither maximum will be used.  I think conversations will be easier if it goes "the default max is 4, if you need to change that, then please set the worker-multiplier".

Reply:
I actually like this approach for simplicity. I will wait for other feedback and will update the PR next week.